aten/src/ATen/native/quantized/cpu/qnnpack/src/q8gemm/8x8-aarch64-neon.S

```diff
@@ -42,33 +42,37 @@ BEGIN_FUNCTION pytorch_q8gemm_ukernel_8x8__aarch64_neon
 
     # https://developer.arm.com/docs/ihi0055/d/procedure-call-standard-for-the-arm-64-bit-architecture
     # Callee need to save 8-15 vector registers and only the lower 64 bits of each.
+
+    # Load params
+    LDP x16, x8, [sp]
+
+    # a1
+    CMP x0, 2
```

Comment:
Same.  I would have expected a condition code on ADD if that was supposed to be conditional?

Reply:
Same as earlier. Just hidden away. https://github.com/pytorch/pytorch/pull/37626/files/20a19b900e54e205185d5ce54a3ec327e54e12c2#diff-f682e6802902a89ec92f72efe5ec2a5bR120

Reply:
Likewise.  It's should be fine to move the ADD as far as I can tell, but moving the CMP is a hard to find bug in the making. :)